HB billing and sub (#18829)
docker/docs: content/billing/build-billing.md

---
title: Docker Cloud Build billing
description: Learn how to buy a Docker Cloud Build subscription.
keywords: payments, billing, subscription, build, cloud, cloud build, remote builder
---

> Cloud Build is a next-gen build tool that enables improved build speed, a shared build cache, and native multi-platform builds. Cloud Build is available for free with an existing Docker subscription. To buy additional seats and increase your available build minutes, you can buy a subscription to enhance your builds.

Reviewer: ```suggestion
Cloud Build is a next-gen build tool that enables improved build speed, a shared build cache, and native multi-platform builds. Cloud Build is available for free with an existing Docker subscription. To get additional seats and increase your available build minutes, you can buy a subscription to enhance your builds.
```

> In this section, learn how to set up and manage billing for your Cloud Build subscription. To learn about the subscription tiers, see [Cloud Build subscriptions and features](../subscription/build-details.md).

Reviewer: suggestion

```suggestion
In this section, learn how to set up and manage billing for your Cloud Build subscription. For more information about subscription tiers, see [Cloud Build subscriptions and features](../subscription/build-details.md).
```

3. Review the plans, then select **Buy now**.
4. For organizations: Select the number of seats you want to buy, then select **Continue to payment**.
5. Review your order, then enter your payment information and select **Subscribe**.

Subscriptions charge monthly to the default payment method.

## View billing history and manage your billing information

You can view your billing history at any time.

1. Sign in to your Docker account and go to the [Docker Cloud Build dashboard](https://www.build.docker.com).
2. Select **Settings and billing**.
3. Select the **Payment methods and billing history** link to go to the billing portal.

Here you can find your invoice history, download your invoices, and manage your billing information and payment methods. You can also find your next billing date.

## Downgrade your subscription

You can downgrade your subscription at any time.

1. Sign in to your Docker account and go to the [Docker Cloud Build dashboard](https://www.build.docker.com).
2. Select **Settings and billing**.
3. Select **Change plan**.
4. Select **Downgrade**.
5. Follow the on-screen instructions then select **Confirm downgrade**.

> Your subscription will remain active until your next billing cycle.

Reviewer: suggestion

```suggestion
Your subscription remains active until your next billing cycle.
```
